lldb/source/Plugins/ObjectFile/PECOFF/ObjectFilePECOFF.cpp

```diff
@@ -1015,7 +1015,7 @@ SectionType ObjectFilePECOFF::GetSectionType(llvm::StringRef sect_name,
           // .eh_frame can be truncated to 8 chars.
           .Cases(".eh_frame", ".eh_fram", eSectionTypeEHFrame)
           .Case(".gosymtab", eSectionTypeGoSymtab)
-          .Case(".swift_ast", eSectionTypeSwiftModules) // downstream change
+          .Case("swiftast", eSectionTypeSwiftModules) // downstream change
```

Comment:
You may need to make this change in a couple more places, including the Swift repo